client/app/components/dynamic-form/DynamicForm.jsx

```diff
@@ -77,7 +77,15 @@ class DynamicForm extends React.Component {
   handleSubmit = (e) => {
     this.setState({ isSubmitting: true });
     e.preventDefault();
+
     this.props.form.validateFieldsAndScroll((err, values) => {
+      Object.entries(values).forEach(([key, value]) => {
+        const initialValue = this.props.fields.find(f => f.name === key).initialValue;
+        if ((initialValue === undefined || initialValue === '') && value === '') {
```

Comment:
What if `initialValue` was `null`? Maybe some `isEmpty` might work better here?

Reply:
I decided to be a bit verbose and check for `null` explicitly because `isEmpty` has issues with numbers.